### PR TITLE
Support for Basque language

### PR DESCRIPTION
**Pull request by iagobaapellaniz**
It needs a bit more investigation. For instance `Figure 1` should be spelled `1. Irudia`. Also hyphenation has to be investigated a bit. It shares a lot of features with Spanish. But for the moment is more than enough.

### CODE REVIEW

File: crates/typst-library/translations/eu.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+figure = Irudia
+table = Taula
+equation = Ekuazioa
+bibliography = Bibliografia
+heading = Atala
+outline = Aurkibidea
+raw = Kodea
+page = Orria
```
Comment (laurmaedje) [inline crates/typst-library/translations/eu.txt:8]: In the [Spanish translation](https://github.com/typst/typst/blob/main/crates/typst-library/translations/es.txt) "page" is written in lowercase. Should that be the case here as well?

Comment (iagobaapellaniz) [reply crates/typst-library/translations/eu.txt:8]: Could be. It depends if it is at the beginning of a sentence. The same for section, raw, and everything else.

Besides, in Spanish raw is also poorly translated. I plan to fix it.

Comment (laurmaedje) [reply crates/typst-library/translations/eu.txt]: > It depends if it is at the beginning of a sentence. The same for section, raw, and everything else.

It's not possible to distinguish for now. You should assume that it's used within the sentence. At the start of the sentence would be less common and needs to be handled manually.

Comment (iagobaapellaniz) [reply crates/typst-library/translations/eu.txt]: It's OK then. I think the PR is ready for supporting basque. I would investigate on the hyphen part later. Thanks for the comments.

**Comment by peng1999** [comment]
> For instance `Figure 1` should be spelled `1. Irudia`.

Related #5102

**Comment by iagobaapellaniz** [comment]
> Related #5102

Indeed the period after is somehow needed in the basque grammar. The period means 1st, 2nd and so on. If not included, a space could be enough but it is weird. On the other hand, a space works better for other counters like `(1) Ekuazioa` or `IV Atala`.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**